home_assistant/server.sls

```diff
@@ -5,13 +5,21 @@ home_assistant_packages:
   pkg.installed:
   - names: {{ server.pkgs }}
 
+home_assistant_user:
+  user.present:
+  - name: home_assistant
```

Comment:
I would prefer defining this username in map.jinja and using `server.user` instead to allow user to override it's name.
Also I would set default to something without underscore: `hass` or `home-assistant` (even dash is better in username than underscore)

Reply:
I could make that change, however the reason I didn't is because it's already `home_assistant` in the existing master branch version and I didn't want to break existing deployments. The only reason these are showing up as new lines is because I moved the stanza closer to the top of the file.